debian/jitsi-meet-web-config.postinst

```diff
@@ -98,7 +98,7 @@ case "$1" in
         # jitsi meet
         JITSI_MEET_CONFIG="/etc/jitsi/meet/$JVB_HOSTNAME-config.js"
         if [ ! -f $JITSI_MEET_CONFIG ] ; then
-            cp /usr/share/doc/jitsi-meet-web-config/config.js $JITSI_MEET_CONFIG
+            cp /usr/share/jitsi-meet-web-config/config.js $JITSI_MEET_CONFIG
```

Comment:
is interface_config already there?

Reply:
Nope, its in the main jitsi-meet folder with rest of the js.

Reply:
👍